lib/kube/proxy/sess.go

```diff
@@ -1017,10 +1018,17 @@ func (s *session) join(p *party, emitJoinEvent bool) error {
 	if p.Mode == types.SessionPeerMode {
 		s.io.AddReader(stringID, p.Client.stdinStream())
 	}
-
 	s.io.AddWriter(stringID, p.Client.stdoutStream())
-	s.BroadcastMessage("User %v joined the session with participant mode: %v.", p.Ctx.User.GetName(), p.Mode)
```

Comment:
Moved so that the new participant doesn't see the redundant broadcast, e.g.:

```
Teleport > User matt joined the session with participant mode: moderator.
Teleport > Joining session with participant mode: moderator
```